### DISCUSSION TITLE
Typing "shift + {" will open the DevTool automatically, typing "t" in textbox on the web page will refresh the web page automatically

### DISCUSSION THREAD

**Discussion by xiedongweo** [cefsharp/CefSharp#4875] [General]
### Is there an existing issue for this?

- [X] I have searched both open/closed issues, no issue already exists.

### CefSharp Version

92.0.260

### Operating System

Windows 11

### Architecture

x64

### .Net Version

.Net4.6

### Implementation

WinForms

### Reproduction Steps

```
public partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();
            var browser = new ChromiumWebBrowser("https://cn.bing.com/");
            browser.KeyboardHandler = new PreviewKeyboardHandler();
            this.Controls.Add(browser);
            browser.Dock = DockStyle.Fill;
        }
    }

    public class PreviewKeyboardHandler : IKeyboardHandler
    {

        public PreviewKeyboardHandler()
        {
        }


        public bool OnKeyEvent(IWebBrowser chromiumWebBrowser, IBrowser browser, KeyType type, int windowsKeyCode, int nativeKeyCode, CefEventFlags modifiers, bool isSystemKey)
        {
            return false;
        }

        public bool OnPreKeyEvent(IWebBrowser chromiumWebBrowser, IBrowser browser, KeyType type, int windowsKeyCode, int nativeKeyCode, CefEventFlags modifiers, bool isSystemKey, ref bool isKeyboardShortcut)
        {
            if (!chromiumWebBrowser.IsBrowserInitialized)
            {
                return false;
            }

            switch ((Keys)windowsKeyCode)
            {
                case Keys.F12:

                        browser.ShowDevTools();
                    break;
                case Keys.F5:
                    if (modifiers == CefEventFlags.ControlDown)
                    {
                        // ctrl + F5
                        browser.Reload(true);
                    }
                    else
                    {
                        // F5
                        browser.Reload();
                    }
                    break;
            }

            return false;
        }
}
```

### Expected behavior

1. typing "shift + {", I want the "{" occurred in the textbox, instead of openning the Devtool.
2. typing "t", I want the "t" character occurred in the textbox, instead of reloading the web page.

### Actual behavior

1. typing "shift + {", the "{" doesn't occurred in the textbox, but it opened the Devtool.
2. typing "t", the "t" character doesn't occurred in the textbox, but it reloaded the web page.

### Does this problem also occur in the CEF Sample Application

Not Tested

**Comment by amaitland** [comment]
> Please only open an issue if you can reproduce the problem with version 122.1.120 or later.

As per https://github.com/cefsharp/CefSharp/blob/master/.github/ISSUE_TEMPLATE/bug_report.yml

> ### CefSharp Version
> 92.0.260

Unsupported. Try the current supported version (M126 at time of writing)

**Reply by xiedongweo** [reply]
I've tested the latest version code sample, the same issue occurred.

**Reply by amaitland** [reply]
Ask on https://www.magpcss.org/ceforum/index.php for help.

`CefSharp` is just a wrapper.
